crates/pallet-subspace/src/lib.rs

```diff
@@ -388,9 +388,10 @@ pub mod pallet {
     #[pallet::storage]
     pub(super) type BlockList<T> = StorageMap<_, Twox64Concat, FarmerPublicKey, ()>;
 
-    /// Mapping from segment index to corresponding Merkle Root.
+    /// Mapping from segment index to corresponding merkle tree root of segment records.
     #[pallet::storage]
-    pub(super) type MerkleRootsBySegmentIndex<T> = StorageMap<_, Twox64Concat, u64, Sha256Hash>;
+    #[pallet::storage_prefix = "MerkleRootsBySegmentIndex"]
```

Comment:
Not sure if this is really necessary, can be removed when we're fine to purge the chain and restart the network.

Reply:
Not necessary, we are at the stage where we break things and reset the network anyway

Reply:
Perfect.